hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -1434,6 +1439,10 @@ public static ClientProtos.Result toResult(final Result result, boolean encodeTa
     builder.setStale(result.isStale());
     builder.setPartial(result.mayHaveMoreCellsInRow());
 
+    if (result.getMetrics() != null) {
```

Comment:
Seems like the [result metrics will get lost for `exists` calls](https://github.com/apache/hbase/pull/6623/files#diff-32e244a21a1e538afa88588f8ced6ff24a107fd5aac01038c528d3d27ac844dbR1454-R1460); thoughts on handling that more explicitly? Maybe throwing an error if someone tries to enable metrics on an Get with `checkExistenceOnly == true`